cabal.project

```diff
@@ -129,6 +135,8 @@ constraints: base16-bytestring <1
 constraints: prettyprinter <1.6.1
 constraints: hashable <1.3.1
 constraints: base64-bytestring <1.1
+constraints: contiguous == 0.6.2.0
+constraints: run-st == 0.1.1.0
```

Comment:
The constraints above are required to not break pact validation. Is that also the case for these?

Otherwise, could you add a comment about why these constraints are needed.

Reply:
It's not the same case. I'll add a comment and put them in their own section.